components/proxy/conf/Caddyfile

```diff
@@ -127,7 +127,6 @@
 }
 
 (server_public_api) {
-	uri strip_prefix /public-api
 	# TODO(ak) verify that it only enabled for json content-type, not grpc
```

Comment:
@akosyakov Since we tested that it works with gRPC now: Can we remove this comment? Or did you plan to test sth else as well?

Reply:
let's remove